src/pages/AgentNew.tsx

```diff
@@ -25,6 +46,23 @@ export default function AgentNew() {
     setOfficeSearchText('')
   }
 
+  const onSubmit: SubmitHandler<IAddAnAgentForm> = async ({
+    office,
+    ...params
+  }: IAddAnAgentForm) => {
+      const newAgent = await ApiAgent.create({
+        ...params,
+        officeId: office.id,
+      })
+
+      if (newAgent) {
+        toast.success('Agent created')
+        navigate(`/agents/${newAgent.agent.id}`)
```

Comment:
Once we navigate to the Agent page, we don't want the "back" button to send them back here, so lets add `replace: true`
```suggestion
        navigate(`/agents/${newAgent.agent.id}`, { replace: true })
```